src/label.rs

```diff
@@ -168,7 +168,6 @@ mod tests {
     assert_ne!(tags[0].label, tags[1].label);
   }
 
-
```

Comment:
Weird to see formatting changes like this in a codebase that is entirely autoformatted with `cargo fmt`. But I'm glad it decided to remove this extra line break today.